_rules/SC1-2-1+SC1-2-3-video-transcript.md

```diff
@@ -24,11 +24,11 @@ authors:
 
 ### Applicability
 
-The rule applies to any [non-streaming][] video element [visible in the page][]
+The rule applies to any [non-streaming][] `video` element [visible on the page][]
```

Comment:
I think this rule should be taken out of the rule group / composed rule that the other rules are in, since the Applicability is quite distinct (videos with/without audio), and they don't even map to the same accessibility requirement. 
With the new split between atomic/composed rules in the ACT Rules Format (see https://www.w3.org/TR/act-rules-format/#rule-types and https://www.w3.org/TR/act-rules-format/#structure-accessibility-requirements), I think it is even clearer that this rule should be an atomic rule on its own.